package proto_ff;

import "yd_fieldoptions.proto";


message E_AttributeAttribute
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "属性ID"];
	optional int32 m_percent = 2[(yd_fieldoptions.field_cname) = "百分比"];
	optional float m_power = 3[(yd_fieldoptions.field_cname) = "战斗力"];
	optional int32 m_systemType = 4[(yd_fieldoptions.field_cname) = "模块类型"];
}


message Sheet_AttributeAttribute
{
	repeated E_AttributeAttribute E_AttributeAttribute_List = 1[(yd_fieldoptions.field_arysize)=256];
}

message E_AttributeSuppress
{
	optional int32 m_lvlDiff = 1[(yd_fieldoptions.field_cname) = "等级差"];
	optional int32 m_hitSupPve = 2[(yd_fieldoptions.field_cname) = "命中压制"];
	optional int32 m_critSupPve = 3[(yd_fieldoptions.field_cname) = "暴击压制"];
	optional int32 m_killSupPve = 4[(yd_fieldoptions.field_cname) = "必杀压制"];
	optional int32 m_stateSupPve = 5[(yd_fieldoptions.field_cname) = "状态压制"];
	optional int32 m_lvSupPve = 6[(yd_fieldoptions.field_cname) = "等级压制"];
	optional int32 m_hitSupPvp = 7[(yd_fieldoptions.field_cname) = "命中压制"];
	optional int32 m_critSupPvp = 8[(yd_fieldoptions.field_cname) = "暴击压制"];
	optional int32 m_killSupPvp = 9[(yd_fieldoptions.field_cname) = "必杀压制"];
	optional int32 m_stateSupPvp = 10[(yd_fieldoptions.field_cname) = "状态压制"];
	optional int32 m_lvSupPvp = 11[(yd_fieldoptions.field_cname) = "等级压制"];
}


message Sheet_AttributeSuppress
{
	repeated E_AttributeSuppress E_AttributeSuppress_List = 1[(yd_fieldoptions.field_arysize)=512];
}

message E_AttributePowersup
{
	optional int32 m_powerDiff = 1[(yd_fieldoptions.field_cname) = "战力差"];
	optional int32 m_powerSupPve = 2[(yd_fieldoptions.field_cname) = "战力压制"];
	optional int32 m_powerSupPvp = 3[(yd_fieldoptions.field_cname) = "战力压制"];
}


message Sheet_AttributePowersup
{
	repeated E_AttributePowersup E_AttributePowersup_List = 1[(yd_fieldoptions.field_arysize)=128];
}
